client/app/lib/realtimecontroller.coffee

```diff
@@ -61,58 +66,52 @@ module.exports = class RealtimeController extends KDController
     # https://www.pivotaltracker.com/story/show/88210800
     @pbNotification = PUBNUB.init options
 
+    fetchServerTime = (callback = kd.noop) =>
 
-  syncTime: ->
+      @pubnub.time (timestamp) =>
+        return callback Date.now() * 10000  unless timestamp
```

Comment:
Would it have any side-effect to set browser time as `@lastSeenOnline` property?